components/sync/driver/brave_sync_auth_manager_unittest.cc

```diff
@@ -47,16 +47,16 @@ class BraveSyncAuthManagerTest : public testing::Test {
   using CredentialsChangedCallback =
       SyncAuthManager::CredentialsChangedCallback;
 
-  BraveSyncAuthManagerTest() : identity_env_(&test_url_loader_factory_) {}
+  BraveSyncAuthManagerTest()
+      : identity_manager_(new signin::BraveIdentityManager()) {}
```

Comment:
This tests nothing about this PR, we should at least make sure `OnAccountsInCookieUpdated` doesn't get called when call `ProfileSyncService::OnEngineInitialized`.

Reply:
Fixed in https://github.com/brave/brave-core/pull/7346/commits/6cfcea34147e6e9873f9c013497bd5625b111920